ISSUE_TEMPLATE/membership.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Organization Membership Request
+about: Request membership in the OpenFeature Org
+title: 'REQUEST: New membership for <your-GH-handle>'
+labels: area/github-membership
+assignees: ''
```

Comment:
Do you know if it's possible to set a group as a default assignee? If so, I think we should use `@open-feature/maintainers`.

Reply:
I think it is a setting we can add after

Reply:
It is possible yes in the CODEOWNERS file

Reply:
ok then if we approve this, we can add the details

Reply:
can be done in the settings once we have created the template